explorer/src/theme.ts

```diff
@@ -1,62 +1,111 @@
-import { createTheme } from '@mui/material/styles';
+// import * as React from 'react';
+// import { Theme } from '@emotion/react';
+// import { PaletteMode } from '@mui/material';
+// import { createTheme } from '@mui/material/styles';
+// import { ExplorerContext, ExplorerProvider } from './context/main';
+// const { mode }: any = React.useContext(ExplorerContext);
 
-const nymPalette = {
-  primary: {
-    main: '#F2F2F2', // white
-    light: 'white', // tbc
-    dark: '#070B15', // Black [top nav bg]
-    selectedText: '#FB6E4E', // orange selected text
-  },
-  secondary: {
-    main: '#009FA8',
-    light: '#5C616D', // grey [hamburger grey]
-    dark: '#242C3D', // lighter black [nav bg]
-  },
-  background: {
-    main: '#242C3D',
-    darkBlack: '#070B15',
-    lightBlack: '#242C3D',
-  },
-};
+// export const theme = createTheme({
+//   palette: {
+//     mode,
+//   },
+// });
+// const nymPalette = {
+//   // DARK THEME
+//   primary: {
+//     main: '#F2F2F2', // white, font, text, main text color
+//   },
+//   // LIGHT THEME
+//   secondary: {
+//     main: '#fff', // white, font, text, main text color
+//   },
+//   background: {
+//     main: '#242C3D',
+//   },
+// };
+
+// const { mode }: any = React.useContext(ExplorerContext);
+
+// export const theme = createTheme({
+//   palette: {
+//     mode,
+//   },
+// });
 
 // A custom theme for this app
-export const theme = createTheme({
-  typography: {
-    fontFamily:
-      'open sans, sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Helvetica Neue", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol"',
-    fontSize: 14,
-    fontWeightBold: 600,
-  },
-  palette: {
-    primary: {
-      main: nymPalette.primary.main,
-      light: nymPalette.primary.light,
-      dark: nymPalette.primary.dark,
-      contrastText: nymPalette.primary.selectedText,
-      // only way to add an additional colour seems
-      // to be `contrastText`?
-    },
-    secondary: {
-      main: nymPalette.secondary.main,
-      light: nymPalette.secondary.light,
-      dark: nymPalette.secondary.dark,
-    },
-  },
-  shape: {
-    borderRadius: 24,
```

Comment:
Might be a good idea to make the `borderRadius` be a multiple of the spacing? (if it isn't already)

Reply:
The theme as-is is commented out actually. Again, not sure if this is an old commit or something you're looking at?

Theme is currently defined in the AppWrapper. It will be broken out again though. This happened as part of the Dark mode refactor.